application/config.json.template

```diff
@@ -86,6 +86,10 @@
         "wsf",
         "wsh"
     ],
+    "githubReferencingEnabledChannelPattern": "general",
+    "githubRepositories": [
+        <repository id here (e.g. 403389278)>
+    ],
```

Comment:
when u do config changes, put it in big fat bold into the PR description. otherwise i wont notice when merging and during release and we have a downtime.

Reply:
### Big fat bold texting incoming....